notebooks/Imputation_best_practices/Imputation_best_practices.ipynb

```diff
@@ -0,0 +1,4557 @@
+{
```

Comment:
<p>Could you add a section here that goes over how these results should be interpreted? Like what is the takeaway? Why are the first 3 rows nan for Mean Filling method?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/data-science-workflows/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
<p>Idea here was to compare the final result of KNN and Mean Imputation(MI) using error and variance that which is giving better results. I understand that the way of presentation is not good. I'll change it to multi table comparison side by side. The first 3 rows for MI is nana because I am using pd.concat which is appending the rows of one dataframe after another. Multi-table comparison will resolve this issue</p>